docs/excel/custom-functions-naming.md

```diff
@@ -0,0 +1,38 @@
+---
+ms.date: 02/08/2019
+description: Learn requirements for Excel custom functions' names and avoid common naming pitfalls.
+title: Naming guidelines for custom functions in Excel (preview)
+localization_priority: Normal
+---
+# Naming guidelines
+
+A custom function is identified by an `id` and `name` property in the JSON metadata file. The function `id` is used to uniquely identify custom functions in your JavaScript code. The function `name` is used as the display name that appears to a user in Excel. A function name can differ from the function ID, such as for localization purposes. But in general it should stay the same as the ID if there is no compelling reason for them to differ.
+
+Function names and function IDs share some common requirements:
+
+- They must only use alphanumeric characters (including Unicode), the numbers zero through nine, underscores, and periods.
+
+- They must start with a letter and have a minimum limit of three characters.
+
+Most function names for built-in Excel functions appear in uppercase letters (such as `SUM`). As a best practice, we recommend to use uppercase for your custom function names so they appear consistent for users.
+
+Function names should not be named the same as:
+
+- Any cells between A1 to XFD1048576 or any cells between R1C1 to R1048576C16384.
+
+- Any Excel 4.0 Macro Function (such as `RUN`, `ECHO`).  For a full list of these functions, see [this article](https://www.microsoft.com/en-us/download/details.aspx?id=1465).
```

Comment:
```suggestion
- Any Excel 4.0 macro function (such as `RUN`, `ECHO`).  For a full list of these functions, download [Macro Function Help File for Excel 4.0 Macros](https://www.microsoft.com/en-us/download/details.aspx?id=1465).
```

Reply:
As far as I know, this is the only list that Excel provides is in this downloadable file?